iphone/Maps/UI/PlacePage/PlacePageInteractor.swift

```diff
@@ -242,6 +238,13 @@ extension PlacePageInteractor: PlacePageHeaderViewControllerDelegate {
   func previewDidPressExpand() {
     presenter?.showNextStop()
   }
+
+  func previewDidPressShare(from sourceView: UIView) {
+    if let shareVC = ActivityViewController.share(forPlacePage: placePageData),
+       let mvc = MapViewController.shared() {
```

Comment:
It would be more readable on the same line, or at least with 4 spaces indent, not 2 spaces.